### DISCUSSION TITLE
Inconsistent mapping of keys to organ

### DISCUSSION THREAD

**Discussion by emseedee** [GrandOrgue/grandorgue#895] [Help] [answered]
I have 2 separate systems with GO installed. On my VPO system, I am running v0.3.1.2334+155 under Ubuntu 18.04 with 2 M+P through a MidiBoutique HWCE and a  Keystation 61ES as a third manual. On my development box I am running 3.3.0-1 under Windows 10 with a single keystation 61ES.

Using the same ODF, when I map the keyboard to the GO on my development box, everything is fine, but on the VPO  there is a one note shift in all the keyboards - i.e. C on the keyboard maps to C# on the VPO. (I've just checked, and I'm getting the same issues with other sample sets, including the demo). 

I know that I can work around this with the transpose option in the keyboard mapping dialog, but is there a setting on GO that I might have inadvertently changed?

Thanks
Mike

**Comment by oleg68** [comment, marked as answer]
![transp](https://user-images.githubusercontent.com/11586506/144489045-b4056b78-9509-4a29-a7ee-d7087a5e7ba1.png)

Check the value in this field. It should be 0.

**Comment by emseedee** [comment]
Got it in one. Thanks very much.

I'd looked at all the menus, but not thought of this

Mike
